backend/routes/XummApi.js

```diff
@@ -0,0 +1,424 @@
+import express from 'express'
+import xrpl from "xrpl";
+import fetch from 'node-fetch'
+import axios from 'axios'
+import {create} from 'ipfs-http-client';
+import XrplNFTHelper from './XrplNFTHelper.js';
+import XummSdk from 'xumm-sdk';
+
+
+const xummApi = ''
+const xummSecret = ''
+
+const pinataApiKey = ''
+const pinataSecretApiKey = ''
+
+const tempAddr = "rfRZeyG8YfSmKPqdX6PVLFJ5bdPCraDAsA"
+
+const minterAddress = "rhk5dvLzcQ9HywR4cTsms3TqunAb94bXmv"
+const minterKey = ""
+
+const sdk = new XummSdk.XummSdk(xummApi,xummSecret)
+
+const router = express.Router()
+
+
+
+
+/*
+* Call to the XUMM sdk. Requires properly constructed payload 
+* and returns data for the user to interact with (.png QR code url).
+*
+* @params: payload object. {TransactionType: "CreateTicket",...}
+* @Returns: XUMM data object. data.refs.qr_png contains qr code information for user to scan with app.
+*/
+const xummCall = async (payload) => {
+
+    const data =  await sdk.payload.create(payload)
+    return data
+}
+
+/*
+* Gets the ticket objects from an account and mints all tickets it finds on the account passed.
+*
+* @params: Post with a body object containing metadata.account. String value with live r-address. 
+* Example: req.body.metadata.account -> {metadata: {account: "rfRZeyG8YfSmKPqdX6PVLFJ5bdPCraDAsA"}  }
+* 
+* @Returns: Updated object with ann array of NFT's for the account. 
+*
+*/
+router.route('/mintTickets').post( (req,res) => { 
+
+  const nftManager = new XrplNFTHelper({Account: req.body.metadata.account})
+  nftManager.getTicketInfo().then( tickets => {
+  nftManager.mintTickets(tickets,minterKey).then(nfts => { 
+
+         res.send(nfts)
+      })
+    })
+})
+
+/*
+* Creates any number of tickets passed. These tickets can later be used to mint in batches greater than 1 via /mintTickets.
+*
+* @params: Post with a body object containing metadata.TicketCount. Integer value greater than 1. 
+* Example: req.body.metadata.TicketCount -> {metadata: {TicketCount: 5 }  }
+*
+* @Returns: XUMM data object. response.data.refs.qr_png contains qr code information for user to scan with app. 
+*/
+router.route('/createTickets').post((req,res) => {
+ 
+  let payload = {
+    TransactionType: "TicketCreate",
+    TicketCount: req.body.metadata.TicketCount

```

Comment:
You should probably add a check here limiting the number of tickets to the max of 250 tickets per account, otherwise giving a nice error. https://xrpl.org/tickets.html#tickets